### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by dumbmatter** [es-tooling/esperf#11]
I ran this on a project of mine and got:

```
▲  eslint-plugin-import - package.json
│  
│  Module eslint-plugin-import could be replaced with a more performant alternative.
│  
│  You can find an alternative in the following documentation:
│  https://github.com/es-tooling/eslint-plugin-depend/blob/main/docs/rules/eslint-plugin-import.md
│
▲  eslint-plugin-react - package.json
│  
│  Module eslint-plugin-react could be replaced with a more performant alternative.
│  
│  You can find an alternative in the following documentation:
│  https://github.com/es-tooling/eslint-plugin-depend/blob/main/docs/rules/eslint-plugin-react.md
```

Both of those URLs are 404 errors

**Comment by 43081j** [comment]
Good catch. This is bad copy paste on my part 

It should point at the module-replacements repo

**Comment by 43081j** [comment]
Fixed by #12
